src/swsssdk/dbconnector.py

```diff
@@ -3,10 +3,61 @@
 """
 from . import _connector_map, logger
 from .interface import DBInterface
-
+import os
+import json
 
 # FIXME: Convert to metaclasses when Py2 support is removed. Metaclasses have unique interfaces to Python2/Python3.
 
+class SonicDBConfig(object):
+    SONIC_DB_CONFIG_FILE = "/var/run/redis/sonic-db/database_config.json"
+    _sonic_db_config_init = False
+    _sonic_db_config = {}
+
+    @staticmethod
+    def load_sonic_db_config(sonic_db_file_path=SONIC_DB_CONFIG_FILE):
```

Comment:
>load_sonic_db_config [](start = 8, length = 20)

I guess you could rewrite _load_connector_map instead of creating a new function.

Reply:
done